org-vet/org-vet.schema.json

```diff
@@ -0,0 +1,130 @@
+  "version": "1.0.0",
+  "required": [
+    "v",
+    "d",
+    "i",
+    "ri",
+    "s",
+    "a",
+    "r"
+  ],
+  "properties": {
+    "v": {
+      "description": "Version string using ACDC conventions",
+      "type": "string"
+    },
+    "d": {
+      "description": "SAID of the credential",
+      "type": "string"
+    },
+    "i": {
+      "description": "Identifier of the issuer",
+      "type": "string"
+    },
+    "ri": {
+      "description": "Issuer's ACDC status registry",
+      "type": "string"
+    },
+    "s": {
+      "description": "SAID of this schema",
+      "type": "string"
+    },
+    "a": {
+      "$id": "EOt6UyUDfF4Ocw1UHAI8rhEQIHOTswqNS2sT2zaXreI6",
+      "description": "Attributes block",
+      "type": "object",
+      "required": [
+        "d",
+        "dt"
+      ],
+      "properties": {
+        "d": {
+          "description": "SAID of attributes block",
+          "type": "string"
+        },
+        "dt": {
+          "description": "Issuance date (when the ACDC was signed).",
+          "format": "date-time",
+          "type": "string"
+        },
+        "loa": {
+          "description": "Level of assurance.",
+          "type": "number"
+        },
+        "des": {
+          "description": "A human-friendly description of the content.",
+          "type": "string",
+          "examples": [
+            "Photo of the left front bumper of vehicle 1 from collision: a 2018 Ford F-150 XLT"
+          ]
+        },
+        "typ": {
+          "description": "The IANA content type of the cited content. This may help a verifier decide whether and how to download and process content.",
+          "type": "string",
+          "examples": [
+            "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+            "image/jpeg"
+          ]
+        },
+        "lbl": {
+          "description": "A convenient label or recommended filename for the content. This may help a verifier match the cited content to its metadata in an original context, but has no defined meaning with respect to how the ACDC is verified.",
+          "type": "string",
+          "examples": [
+            "invoice-03-25-2022.docx",
+            "Post on X on 25 July 2025"
+          ]
+        },
+        "content": {
+          "oneOf": [
+            {
+              "description": "A SAID for the content section. This is not the same as a SAID for the pure content bytes, because it treats the content as a base64url JSOn string, and includes in the hash input a salty nonce.",
+              "type": "string",
+              "examples": [
+                "EK2r6EnDXre2pecTBO8s99j4OtNaaDIhVyr7uGugDhmp"
+              ]
+            },
+            {
+              "$id": "EFSLhiSLGmL-jp0sWcoDh2_05o4tO8dKJXY10EpXPjcT",
+              "description": "content block",
+              "type": "object",
+              "required": [
+                "d",
+                "u",
+                "cargo"
```

Comment:
I don't see a description for this property. Is this supposed to be optional?

Reply:
This is weird. I thought I deleted it. My code is different from this.